scripts/sysman/frmwareSecurityVersion.yml

```diff
@@ -0,0 +1,59 @@
+#
+# Copyright (C) 2023 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+#
+# See YaML.md for syntax definition
+#
+--- #----------------------------------------------------------------------------------
+type: header
+desc: "Intel $OneApi Level-Zero Sysman Extension APIs for firmware security version"
+version: "1.9"
+--- #----------------------------------------------------------------------------------
+type: macro
+desc: "Anti roll-back security firmware version"
+version: "1.9"
+name: $S_ANTI_ROLLBACK_SECURITY_VERSION_EXP_NAME
+value: '"$XS_extension_anti_rollback_security_version"'
+--- #--------------------------------------------------------------------------
+type: enum
+desc: "Anti rollback security version Extension Version(s)"
+version: "1.9"
+name: $s_firmware_security_exp_version_t
+etors:
+    - name: "1_0"
+      value: "$X_MAKE_VERSION( 1, 0 )"
+      desc: "version 1.0"
+--- #--------------------------------------------------------------------------
+type: function
+desc: "Get anti roll-back security firmware version number of the current running firmware"
+version: "1.9"
+class: $sFirmware
+name: GetSecurityVersionExp
+details:
+    - "The application may call this function from simultaneous threads."
+    - "The implementation of this function should be lock-free."
+params:
+    - type: $s_firmware_handle_t
+      name: hFirmware
+      desc: "[in] Handle for the component."
+    - type: char
+      name: "version[$S_STRING_PROPERTY_SIZE]"
+      desc: "[out] NULL terminated string value. The string \"unknown\" will be returned if this property cannot be determined."
+--- #--------------------------------------------------------------------------
+type: function
+desc: "Set anti roll-back security firmware version number"
+version: "1.9"
+class: $sFirmware
+name: SetSecurityVersionExp
+details:
+    - "The application may call this function from simultaneous threads."
+    - "The implementation of this function should be lock-free."
+params:
```

Comment:
Are we sure that this function implementation should be lock free?  while setting SVN via IGSC, we might need lock right? @aravindksg  @joshuaranjan  @AshwinKumarKulkarni   @bellekal

Reply:
I think it says **should** and not **must**. From https://spec.oneapi.io/level-zero/latest/core/INTRO.html#terminology **should** means it is recommended and not mandated. So i think it is ok to use "should be lock free".